assets/js/components/data-block.js

```diff
@@ -71,15 +72,14 @@ class DataBlock extends Component {
 		} = this.props;
 
 		const role = ( 'button' === context ) ? 'button' : '';
-
 		return (
 			<div
-				className={ `
-					googlesitekit-data-block
-					googlesitekit-data-block--${ context }
-					${ selected ? 'googlesitekit-data-block--selected' : '' }
-					${ className }
-				` }
+				className={ classnames(
+					'googlesitekit-data-block',
+					className,
```

Comment:
You previously mentioned that changing the order of these class names messed with the CSS specificity for some of the classes (see: https://github.com/google/site-kit-wp/pull/1105/files#r377040444).

Was that not the case after all or is this an oversight? I thought the order here needed to be preserved—if that isn't the case: great! But could you elaborate on what the issue was/what you thought it was and whether this is an intentional/okay change to make after all? 😄

Reply:
When I initially did that it broke and I had layouts getting messed up. @aaemnnosttv pointed out it shouldn't happen. I think it's also because tests were failing after I reordered initially I thought it was specificity. I was wrong. ![doh!](https://user-images.githubusercontent.com/1387263/74574591-7d9dea80-4f84-11ea-952d-7f23caa3c565.png)

Reply:
Ah, awesome! Good to know, as it's always a pain redoing weird specificity rules. Sounds good and like this is sorted, thanks for the explanation 👍